public/manifest.json

```
@@ -1,7 +1,7 @@
 {
   "manifest_version": 3,
   "name": "Element Searcher",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

This is a feature so I suggest a minor version bump

```suggestion
  "version": "1.2.0",
```